### PR TITLE
editoast: move ElectricalProfileSet to modelv2

### LINKED ISSUE #6739: move ElectricalProfileSet to modelv2

### PR DESCRIPTION
**Pull request by hamz2a**
closes #6739

### CODE REVIEW

File: editoast/src/tests/electrical_profile_set.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "id": null,
+  "id": 0,
```
Comment (leovalais) [inline editoast/src/tests/electrical_profile_set.json]:
```suggestion
```

File: editoast/src/fixtures.rs [hunk 2]
```diff
@@ -388,11 +387,12 @@ pub mod tests {
...
         db_pool: Data<DbPool>,
     ) -> TestFixture<ElectricalProfileSet> {
-        TestFixture::create_legacy(
+        TestFixture::create(
             serde_json::from_str::<ElectricalProfileSet>(include_str!(
                 "./tests/electrical_profile_set.json"
             ))
-            .expect("Unable to parse"),
+            .expect("Unable to parse")
+            .into_changeset(),
```
Comment (leovalais) [inline editoast/src/fixtures.rs]: Best to deserialize into a changeset directly (you have `#[model(changeset(derive(Deserialize)))]`).

**Review by leovalais — CHANGES_REQUESTED** [review]

File: editoast/src/fixtures.rs (hunk 2 above)
Comment (hamz2a) [reply editoast/src/fixtures.rs]: Done.

**Review by leovalais — APPROVED** [review]
LGTM, not tested

**Pull request merged**